### ISSUE TITLE
`Table.to_json` appears to ignore modified column names 

### ISSUE THREAD

**Issue by dcloud** [onyxfish/agate#439]
While using `agate==1.2.0`, I imported a csv file where the header for a column was empty. I was able to add a header by using `Table.rename`, but that name disappeared on saving `to_json`. FWIW, `to_csv` handled it without error.

```python
intbl = agate.Table.from_csv('input.csv')
tbl = intbl.rename(column_names=('countryname',) + intbl.column_names[1:])
tbl.to_json('output.json')
```

**Comment by dcloud** [comment]
Looks like maybe other methods ignore the renamed column name as well. Seeing error on `order_by('countryname')` as well.

**Comment by nbedi** [comment]
I believe this is related to a known issue with `Table.rename` (#423). Fix should be in next minor release.

**Comment by onyxfish** [comment]
Yep, reasonably certain this is a dupe of #423. @dcloud would you be be able to try running your script with the master branch to verify it works as expected for you?

**Comment by dcloud** [comment]
Yes, it works as expected when running on master. Thanks and keep up the great work!

Closing as duplicate of closed issue #423.

**Comment by onyxfish** [comment]
Thanks @dcloud, we'll fold this into a release soon!
